ansible.cfg

```diff
@@ -1,9 +1,9 @@
 [defaults]
 
 # Default path to roles directory
-roles_path         = ./roles
+roles_path = ./roles
 
 # Adds timestamp to each task and add a recap on the end of the playbook
 callback_whitelist = profile_tasks
-log_path=tasks/logs/ansible.log
+log_path = ./tests/logs/ansible.log
```

Comment:
I don't think it is ok for normal operation. In case the customer is not running tests why should he look for logs in test folder?